### DISCUSSION TITLE
Status of this repo

### DISCUSSION THREAD

**Discussion by zheyu001** [backube/snapscheduler#142] [Q&A]
Hi there,

We're working on data durability for our project and snapshot scheduler provided by your repo is quite interesting to us. 

But before install this to our prd env, we are wondering if this repo is still actively maintained?

**Comment by JohnStrunk** [comment]
It is still maintained, but I don't have much planed for feature development-- though I'm happy to have contributions.
If it meets your use case, it should be fine to use.

**Reply by muffl0n** [reply]
Is this still the case? We're thinking about using the scheduler, but since the last release is a year old we're not quite sure about the future of this project. Also there are quite some unmerged dependabot PRs.

Don't get me wrong, I don't wanna be a choosing beggar! Just curious if it's safe to jump aboard. :)
